frontend/public/components/markdown-view.tsx

```diff
@@ -89,13 +89,19 @@ export class SyncMarkdownView extends React.Component<
         <link rel="stylesheet" href="${link.href}">`,
       '',
     );
+    const content = this.props.truncateContent
+      ? _.truncate(this.props.content, {
+          length: 100,
+          separator: ' ',
```

Comment:
```suggestion
          separator: ' ',
          omission: '\u2026', // ellipsis character
```

Reply:
[_.truncate](https://lodash.com/docs/4.17.15#truncate) already defaults to the ellipsis character

Reply:
> [_.truncate](https://lodash.com/docs/4.17.15#truncate) already defaults to the ellipsis character

It defaults to `...` which is not the same as the Unicode ellipsis.